src/main/java/com/puppycrawl/tools/checkstyle/meta/MetadataGeneratorUtil.java

```diff
@@ -99,15 +98,4 @@ private static void dumpMetadata(Checker checker, String path) throws Checkstyle
 
         checker.process(validFiles);
     }
-
-    /**
-     * Return all token types present in checkstyle.
-     *
-     * @return list of token type names
-     */
-    public static List<String> fetchAllTokens() {
```

Comment:
found unused method

Reply:
I do not see it at https://github.com/checkstyle/eclipse-cs/pull/257/files also.
only this two plugins could use it.

Reply:
To me it looks like change was in support of the other changes to `DescendantTokenCheck` in the same PR. 
https://github.com/checkstyle/checkstyle/commit/39b02e16d55a081ba779c0bdb2318ab30eaea515#diff-e560d85ecb7f4a0c7b4ee5a5c8384dfc303d255e5aee6591abec990b025340f0R359

This looks familar like something I pointed them to as they seemed to be duplicating existing functionality. There is nothing in PR, so I don't know if it was in gitter or I am imagining it.

Reply:
ok to remove.